File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/UpgradeStorage.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.MainActivity;
import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Upgrade;

/* JADX INFO: loaded from: classes3.dex */
public class UpgradeStorage extends Upgrade {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.upgrade_storage_name;
        this.idDescription = R.string.upgrade_storage_description;
        this.idImage = R.drawable.upgrade_storage;
        this.notSellable = true;
        this.price = 1L;
        this.gemPrice = 80;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Upgrade
    public void use() {
        MainActivity.data.setUpgradeStorage(MainActivity.data.getUpgradeStorage() + 1);
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/UpgradeStorage.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.MainActivity
import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Upgrade

class UpgradeStorage : Upgrade() {
    override fun configureProperties() {
        idName = R.string.upgrade_storage_name
        idDescription = R.string.upgrade_storage_description
        idImage = R.drawable.upgrade_storage
        notSellable = true
        price = 1L
        gemPrice = 80
    }

    override fun use() {
        MainActivity.data.upgradeStorage++
    }
}
